transport/pb/groupcache.proto

```diff
@@ -37,6 +37,15 @@ message SetRequest {
   optional int64 expire = 4;
 }
 
+message RemoveMultiRequest {
```

Comment:
We call it `batch` in some places in the code, and `RemoveKeys` elsewhere. We should be consistent.

```suggestion
message RemoveKeysRequest {
```